app/models/incomeSourceDetails/BusinessNameForm.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.incomeSourceDetails
+
+import play.api.data.Form
+import play.api.data.Forms.{mapping, text}
+
+case class BusinessNameForm(name: String)
+
+
+object BusinessNameForm {
+  val form: Form[BusinessNameForm] = Form(mapping(
```

Comment:
might be better to define validation on the model itself, so we can unit test this easily: "Defining ad-hoc constraints" -> https://www.playframework.com/documentation/2.8.x/ScalaForms